streamly.cabal

```diff
@@ -727,6 +730,12 @@ test-suite Unicode.Stream
   main-is: Streamly/Test/Unicode/Stream.hs
   ghc-options: -main-is Streamly.Test.Unicode.Stream.main
 
+test-suite FileSystem.Handle
```

Comment:
I got confused for a bit there. Maybe you could add `FileSystem.Handle` in the title as well.